### ISSUE TITLE
[Suggestion] Revert changes made to syringes.

### ISSUE THREAD

**Issue by Jakksergal** [VOREStation/VOREStation#3972]
I don't believe that this change accurately represents the wishes of the community and needs to be reverted to its previous state. Not changed to make more viable, not repealed and replaced, reverted. There should be no need to simulate some sort of pseudo-realism in a 2D spessgame.

Since the reason from this change stems from community preference, I'll leave a strawpoll here.
https://www.strawpoll.me/16010811

**Comment by Heroman3003** [comment]
There is a forum thread discussing those changes already.

**Comment by Wickedtemp** [comment]
A forum thread that can now have a strawpoll linked to it, which I'm doing at the moment :p

**Comment by Draycu** [comment]
Honestly?  Reverting would be a better change, and the poll is reflecting that at the moment.  Thanks for making the issue here on Git, Jakk.

**Comment by nerdass** [comment]
why don't we add a research creatable self sterilizing syringe that can do like 30 injections on a charge and can hold bigger reagent containers than 15 units

**Comment by Wickedtemp** [comment]
Why don't we just revert the annoying thing

**Comment by AaronSherwood117** [comment]
I skimmed the thread and popular opinion kind of agrees.  In theory, the change was a nice bit of fluff, but in practice it's just a bunch of extra hassle without any real benefit added.  I don't play medical much anymore, but I'm going to voice support to a revert simply because it's such a common consensus.

**Comment by Little-119** [comment]
The best solution would probably be to change the sprite and(/or) fluff of the syringes to be a future tech self-sterlizing syringe/hypospray or something, but revert their mechanics to the old syringe mechanics. So, you'd get the fluff benefits of the change while not affecting gameplay.

**Comment by nerdass** [comment]
reusing needles is also incredibly gross. like no offense but I don't see why people are upsetti in the spaghetti over the disposable needles being enforced as disposable.

like i don't mean offense but from a roleplay perspective, cross contaminating all your bottles by dipping the syringe into all of them, while not mechanically enforced, is probably something that your characters would all have been taught not to do in med school.

**Comment by SpadesNeil** [comment]
I'm of the opinion that _how about we give the other departments ways to participate?_

A self-cleaning syringe should be super easy for science to make, and so suddenly players from science actually get to do something helpful.

But meanwhile, medical isn't completely fucked because **you guys literally have boxes and boxes of syringes**. How many does each box hold?

**Comment by Draycu** [comment]
What I think you missed is that the fuss is over the infection mechanic that is rather unrealistically implemented right now.  

As soon as you uncap a syringe, it begins building an infection percentage.  It’s not even a matter of injecting a person first, simply drawing medicine starts a timer.

**Comment by nerdass** [comment]
oh, yeah that does sound a bit weird.

**Comment by SpadesNeil** [comment]
Oh. Okay yeah fuck that, it can be done infinitely better and should be reverted until it is done in a way that isn't that stupid.

**Comment by AaronSherwood117** [comment]
Do we make another issue to fix that?  Kinda noticing a theme here.

**Comment by CameronWoof** [comment]
I'm not in favor of making the department a total chore until science comes in and saves the day, because historically, _they never do._ I can count the number of times I've seen the advanced surgery equipment on one hand, and I've straight-up never seen the advanced scanners or rollerbeds outside of the colony map.

**Comment by SpadesNeil** [comment]
>I'm not in favor of making the department a total chore until science comes in and saves the day, because historically, they never do. I can count the number of times I've seen the advanced surgery equipment on one hand, and I've straight-up never seen the advanced scanners or rollerbeds outside of the colony map.

You have a bajillion needles laying around. It's not a chore to just get another needle.

The current mechanics are just stupid. We should be reverting that before we even _consider_ any alternatives.

**Comment by Little-119** [comment]
Add space aids, a persistent disease transmitted by being on the same tile as someone for several minutes, or via reusing syringes.

**Comment by CameronWoof** [comment]
@SpadesNeil That isn't the problem. Medbay players aren't reporting a massive slew of infections despite following the "rules" properly: they're reporting that micromanaging syringes isn't any fun, and the mounting stack of suggestions being made to minimize or outright avoid having to deal with the mechanic entirely is testament to how undesirable this update really is.

**Comment by SpadesNeil** [comment]
>@SpadesNeil That isn't the problem. Medbay players aren't reporting a massive slew of infections despite following the "rules" properly: they're reporting that micromanaging syringes isn't any fun, and the mounting stack of suggestions being made to minimize or outright avoid having to deal with the mechanic entirely is testament to how undesirable this update really is.

>The current mechanics are just stupid, and **start getting infected the moment you open a needle apparently.** We should be reverting that before we even consider any alternatives.

**Comment by nerdass** [comment]
yeah but on the other hand, reusing syringes is disgusting.

**Comment by CameronWoof** [comment]
@SpadesNeil Okay, and that's dumb, but that's not the problem. The problem is that it's tedious to manage, not that there's an unfair time constraint between uncapping them and using them. If you fixed that time constraint, the system would still be tedious and exactly as bad in 99% of cases.

**Comment by SpadesNeil** [comment]
I can think of no reasonable excuse for why you need to be able to re-use syringes, other than powergaming, and sucking meds out of a jar that can hold up to 100 units and fit in your pocket. Furthermore, the idea of re-using needles is exceedingly gross.

What ever happened to pills, anyway?

**Comment by MrSign** [comment]
@SpadesNeil 

Syringes are more 'controlled' doses compared to pills. Considering you can put five units of three different medicines in one go, rather than shoving three different pills down someone's gullet. Chemicals injected directly also act immediately, compared to ingested chemicals which take like... five extra seconds to start working.

I know when I play medical, I mostly focus on pill-usage because the people that come into medical bay when I play are conscious and not in an immediate state of dying. So, I just hand 'em several pills to take and a cup of water to chug 'em all down before patting them on the head and sending them off.

The entire population of the server probably have arms that look like pin cushions! Though, there are times when syringes do outrank pills and that's mostly during expeditions in hostile environments where taking your helmet off = death. IE: Space, Virgo 2, Virgo 3B's phoron atmosphere.

**Comment by Wickedtemp** [comment]
@SpadesNeil 
TLDR: Syringes are better than pills except when pills are better than syringes. They both have their uses, pros, and cons. Going pills-only isn't a good idea. Going syringe-only isn't as bad of an idea but still isn't good. 

Because syringes and pills are different ways to administer medicine that each have their pros and cons.

Your primary damage type healing reagents are going to be primarily administered with syringes because it's standard to make bottles of medicine to put in the medical belt. It's easier on the chemist, too. Injected reagents metabolize at the standard rate and you can control the dosage much more easily in 5u increments, whereas pills are stuck at whatever the chemist made them at. They're also easier for the doctor to administer. If a doctor administers a pill, they get "DOCTOR FORCES PATIENT TO SWALLOW PILL", which implies grabbing the patient, shoving a pill in their mouth and smothering their face and holding their jaw shut until they swallow it. Much worse than a simple injection, which isn't at all odd for a doctor to do. 

Pills, and all other ingested reagents, will metabolize at half the speed, making them slower in general. Pills are generally better for things you'd want in super, super small doses or super high doses, below 5u. (Ryetalyn and Alkysine are two examples of low dose) or above 15u (Tricordrazine, inaprovaline, dylovene, bicaridine and cillin if you're wanting to OD).  Some reagents only work when ingested, such as nutriment, biomass, protein, iron, and carbon. They're also decent for low-priority mixes, such as anti-radiation, or a dylovene/synap mix for anyone hallucinating, it's more efficient than having chem bottles because the pills can hold about as much and can be put into a pill bottle. 

So they both have their uses. Different chems are better used in one than they are in the other, it's one of the neat things about Medical. 

Also, I wouldn't consider using the same syringe as powergaming, since... well... it's something that was always done by just about everybody, on every bay/polaris-code server, up until about 4 months ago. There wasn't anything that said you had to use a new syringe, nobody cared if you did, some people preferred to use new ones but it wasn't like it was rulebreaking if you didn't. Otherwise, just about every single person who played medical would've been getting bwoinked. It was just how the game was played, and now it's being played differently. Doesn't mean the previous way was powergaming. Phrasing it like that makes it sound like nearly every single person that's ever played medical was a constant rulebreaker when all they were really doing was just doing the role as normal.

**Comment by AaronSherwood117** [comment]
I think I'd like a reworked R&D menu before you want to add even more stuff to the list Ace.  We can't even filter by tech or department or even alphabetize.

**Comment by nerdass** [comment]
i mean.

yanno.

you could just HAND the patient the pill.

**Comment by CameronWoof** [comment]
One point at a time, I guess. First of all, implying that the way that syringes work on every server except this one is "powergaming" is an incredibly poor argument. Wanting things to behave intuitively and as you'd expect from years of playing on this and other servers is not powergaming. 

Second, there are ways to make Science "more fun" other than making other departments worse and forcing them to go through Science to be playable again.

**Comment by Cameron653** [comment]
Did some testing.
https://i.imgur.com/m6Zdwd1.png

Results:
Capped (control) syringe: Stayed at 0 dirtiness (infection chance)
Uncapped syringe: Stayed at 0 dirtyness
Uncapped syringe filled w/ fluid: Stayed at 0 dirtyness
Uncapped syringe used to inject someone: Raised to 75 dirtiness

Result: Pulling fluids into syringes doesn't start making the infection chance rise. 


I personally don't like the syringes having an infection chance. It's a bad game mechanic that punishes players for things out of their control. 
Who gets punished by medical doctors using dirty syringes? **The patients do.** They have to take time to go from wherever they're at to go to medical to get a spaceacillin injection.

Does this waste the medical doctor's time? Not really. The MD is most likely in medical, where all they have to do is grab a spaceacillin syringe and inject the person, taking at most a minute of their time. The patient, on the otherhand, **had no control** over being infected whatsoever and had to drop everything & go to medical. 

It's not a fun game mechanic. It punishes everyone ***other*** than medical staff and adds nothing to the game but tediousness.

**Comment by Wickedtemp** [comment]
Can we hear arguements *for* this change? 

They're literally all just "But what about muh realism/science/pills" and "I mean I don't see anything wrong with it." from people who didn't play medical before (or after) the change, so they don't have an idea as to the increase in tediousness the change can bring. I've seen plenty of "okay but i mean is it honestly bad guys", I've seen MANY negative posts about the change, but I don't think I've seen a single argument that directly outlines the reason we need a change that most players hate and would rather not have anyway. 

It isn't even a realistic change. IRL, you're not going to die in fifteen minutes because a nurse couldn't get a blood sample the first time finding a vein so they poked you a couple more times. 

It doesn't add anything to the game except irritation to all parties. Nearly every medical player agrees. "But what about science" - oh, you mean the department that already has a *list* of things to make for Medical? Analyzers, roller beds, laser scalpels, bone clamps, IMS, bags of holding, bluespace beakers, equipment upgrades, Oddy's (Basically a paperweight because mechs need to be redone but thats aside the point), slime jelly and extract, power cells, honestly I'm sure I'm missing something. Science is already extremely relevant to Medical. I get that they don't have stuff to do for the whole six hour round, but *nobody* does, and if they have mining but haven't gotten requests from medical, that doesn't mean they can't make stuff anyway as a surprise. Medical always benefits from the supplies Science produces. 

"But pills" - already went over it in a previous post, the tldr of it being Doctors can't administer pills without an intense RP message and pills are situational anyway in the same way injectors, IV's, hypo's and syringes all are. 

"But medical has plenty of syringes" literally meaningless. Nobody's upset at medical running *out* of syringes. The issue we have is that the primary means of administering chems (which as of now are the only way to actually heal damage in any meaningful capacity for brute/burn and *the* only way for infections, radiation, genetic, toxin and oxy) - was changed to be more tedious for no discernable *purpose*. 

And Cameron's right. Honestly this doesn't really effect a doctor unless they get charged with negligence or fired, which may or may not happen. Maybe eventually they'll get a job ban. But for the duration of the round, the patients are the one who get shafted over this - and this is *why* its so irritating for Medical. A few players don't even play Medical anymore because of this change.

I know, "Well they should get over it", but I'd suggest re-reading it a few more times until it sticks instead of going "lol apathy". Players who used to play Medical frequently no longer do so as frequently in large part due to this change. And thats annoying as hell, because it leads to understaffed Medbay, which means the one or three people that are there end up having to do *everything*, and trust me, you don't want to be the only one or two medics for an entire med-pop round. It can honestly just *suck* sometimes because every time you leave medbay you risk someone running in hoping to find you to help them. If Science blows up, it's all on you to save every patient unless you can find someone to help transit the injured. 

And if the same person has to deal with that frequently, then they'll probably quit because it isn't usually fun to be the only active medic. 

Even if you ignore all of that and go straight back to "nah just get over it", that still doesn't change the fact that, like it or not, the majority of the playerbase hates this enough that they'd rather see it gone, that it provides zero benefit or reason to keep in spite of this, that it wasn't necessary to implement in the first place, and every server on this codebase seems to agree.

And, honestly, if you can't think of any way to make a department like science or cargo more useful aside than "Make X department worse or more tedious and give Sci/Cargo the solution", you need to brainstorm a bit more, or hold more discussion about potential additions.

**Comment by SpadesNeil** [comment]
Okay, you want to know what my argument for the change is? For there to be _some kind of consequence_ for using the same needles over and over?

The same reason mining doesn't start with the best gear.
The same reason why research and development doesn't have their own ore refinery.
The same reason why engineering can't just print new hardsuits out of a machine.
The same reason Security doesn't have its own fully stocked infirmary.

What I want there to be is a consequence for using needles, which are otherwise plentiful, but can ultimately break or cause infections after repeated uses.

#The argument is not a matter of realism. Stop acting like it is.

The argument is a matter of game balance and stopping bullshit like people who carry jars and jars and jars of uber medical mix off on expeditions or just carrying them around in their belt so they are essentially a walking surgery room. It's overpowered as fuck and it's not fun for anyone except the individual.

If we made syringes have limited uses **but then** added something like a self-cleaning syringe _or hypospray_, suddenly the game becomes a lot more interesting for more people.

**Comment by Wickedtemp** [comment]
Alrighty! So now that we have an argument *for* it, things can be discussed in a little more detail. 

As far as I can tell, being someone who's only played Medical exclusively, re-usable syringes haven't been a large contributor in upsetting the balance of medical. Generally, those issues have been tied to things such as one chem healing too much or too little. Chems were never, and still aren't, a total replacement for surgery, rather, they're the only means of healing most types of damage (something that surgery can't even *do* right now) and will soon be getting a nerf for biciaridine, kelotane and dermaline, making them less effective in general while still maintaining usability. The only issues with medical I see are ones we're already working towards resolving - decentralizing Medical from chemistry and giving surgery a couple more uses. 

For this change, in particular, having syringes break after a few uses would be preferred over "use it on literally anybody past the first injection and they could die in fifteen minutes". If the syringe just breaks and becomes unusable without harming the patient, you can get the desired effect of stopping people from re-using the syringes as much, while getting rid of the absolute detriment the current system is for the patients and general crew.

Depending on how it's coded, it may need to be tweaked because having it inject all 15u at once is causing issues, and has been since it was implemented: https://github.com/VOREStation/VOREStation/issues/3543

I recommend going by the amount of a reagent injected rather than a number of injections done. 45-60u would mechanically amount to 3-4 'injections', and it'd mean if this is implemented and we, later on, revert the '15u injection' bit, we wouldn't need to make any changes to the syringe breaking codewise, since if it is done by injection counting, you'll have 3 5u injections per syringe and the number would need to be tweaked to compensate for that. 

And on a more nitpicky note, there really isn't an "uber medical mix" that magically fixes everything. The more you add to a mix, the more everything's diluted, so most mixes are inherently situational. For example, stabilizing mixes are excellent for when you want to stabilize someone out of critical condition, but if you're wanting to heal anything more than that, you'll be injecting the entire bottle for one or two patients. The more widespread the healing is, the more it's diluted. The more you concentrate towards one damage type, the faster it heals, but you get less healing power per bottle. KeloDerma heals faster than Dermaline, but Dermaline will heal more per unit of the mix. Chemistry's great, and it'll still be useful after the adjustments are updated in, but aside from adminodrazine or whatever it is, and perhaps cryo, there isn't an "amazing magical mystery cure-all"

**Comment by AaronSherwood117** [comment]
To sum up the discussion here, please correct me if I'm wrong.

Ace, you want the constant syringe usage to have a penalty because it's one of the things that otherwise has no upgrades because it's the best it can be from the start, and there needs to be SOME downside to shoving a syringe on your ear and drawing heals from your doctor pockets.

Tempest, you're okay with syringes getting SOME sort of nerf from their old state, but the current implementation is way more intrusive than just letting them break.  Such as, changing how injections work in general by not letting you use fixed amounts, and the patient having to pay for the negligence of a medical player.  

Is this about right?

**Comment by SpadesNeil** [comment]
Cameron was the one who originally suggested that needles should be able to break over time. IRL, needles get badly damaged after repeat uses.

And by break I only mean the needle becoming unusable, or possibly it's still usable but causes a small amount of brute damage when used, _eventually_ starts causing infections after _continuing_ to use, etc.

The fine details can be discussed later and aren't required. The end goal is that for some reason or another, needles become unusable after poking them into a person several times.

**Comment by CameronWoof** [comment]
I still don't understand why you feel it's necessary to punish medbay players for playing the department naturally. If the problem is that chems are too strong, there are much less annoying ways to tweak that. Infected/damaged needles are a tax on the _patient's_ time and the doctor's inventory space, but don't make liquid chems any less useful or desirable. I feel that in a few months, when you realize that chem bandoliers are still the standard and people still avoid pills like the plague, things are just going to get even worse.

If you think reusing needles is gross, feel free to set these limitations to yourself. There's no reason to force it on everyone else when we've been absolutely fine imagining that in five hundred years, we've figured out how to make needles that self-sterilize, or out of a material that doesn't degrade.

**Comment by AaronSherwood117** [comment]
I think Ace is being fairly reasonable with this judgement now and we should probably continue this discussion in another place now that there's a proper starting point and common ground.  Git comments aren't really the best format.

**Comment by SpadesNeil** [comment]
I kind of like having the debate here, but the initial goal of this revert thing is now officially decided.

**Needles will have their infection feature reverted.**

The goal was probably to stop medbay players from powergaming by making a mechanic to deter frivolous over-use of the same needle, so you can't just walk around with a dozen 100 unit beakers full of super meds. The solution we got was not ideal, and there's better ways to do it. Diseases are excessively punishing for what was probably intended to be stopped, and the punishment didn't actually affect the doctor who was abusing the system.

**Comment by Wickedtemp** [comment]
It doesn't happen as often as it should, but I have a list that I'll usually send to the RD if there's one around. I know Hemdall and Bratton have both gotten faxes from Tempest with hir list.

**Comment by jakeramsay007** [comment]
So I just want to add in to this; the problem with the PR is not that syringe reuse is a bad idea, it's the fact that after using the syringe once on yourself it starts gaining 'dirtiness' every tick up to 75% chance of infecting yourself upon using the syringe again. Remove the on-tick increase and it's a lot more reasonable.
